deploy/bin/spack-check

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash -eu
+
+CONFIG=$1
+
+. /etc/profile.d/modules.sh
+
+export MODULEPATH=""
+. ${CONFIG}
+
+RELEASE=unstable
+STAGES="libraries applications"
+VENV=$(mktemp -d)/venv
+MAKEFILE=$(mktemp)
+
+echo "Using release: ${RELEASE}"
+
+module load ${RELEASE} python
+
+if [[ ! -d ${VENV} ]]; then
+    python3 -mvenv ${VENV}
+    source ${VENV}/bin/activate
+else
+    source ${VENV}/bin/activate
+fi
```

Comment:
In the current version of the script `VENV` is always a new tmp directory, so we could simplify with
```
    python3 -mvenv ${VENV}
    source ${VENV}/bin/activate
```

Reply:
Done